### ISSUE TITLE
Show OpenSearch version in Dashboards

### ISSUE THREAD

**Issue by macrakis** [opensearch-project/OpenSearch-Dashboards#3050]
The Bug Report template requires the versions of OpenSearch and of OpenSearch Dashboards.

The life buoy icon (is that the new standard for Help?) in the upper right of the Dashboards interface displays "v 2.4.0", which is presumably the OpenSearch Dashboards version. How do I find the underlying OpenSearch version?

**Comment by tmarkley** [comment]
Hey @macrakis, these are good call-outs, thanks!

First, to answer you question about finding the version of OpenSearch: you can find that using the status page (e.g. https://playground.opensearch.org/status) or the status api (e.g. https://playground.opensearch.org/api/status).

I think there are a few follow-ups to make this process smoother for the community:
1. Explicitly state that the version in the help menu is for OpenSearch Dashboards.
2. In the issue template, instead of requesting the version of OpenSearch we can request all of the data sources since [multiple data sources](https://opensearch.org/docs/latest/dashboards/discover/multi-data-sources/) has been released.
3. Provide instructions for finding the versions of data sources. Showing this data in the UI outside of the status page is being discussed over in https://github.com/opensearch-project/OpenSearch-Dashboards/issues/731.

**Comment by joshuarrrr** [comment]
Yeah, I think we can work this backwards a bit. First, installation information is critical and helpful for debugging? Once we've defined that critical data, we need to make it retrievable through a single action in the UI (perhaps by enhancing the existing help menu item or adding a new one). Then we can make sure the template is just a single instruction for how to get the debugging info.

**Comment by douglasg14b** [comment]
Seriously, we've spent the last 30 mins trying to figure this out using the UI alone. Honestly kind of frustrating.
